> @kartben it seems that I don't have the access rights to modify the `MAINTAINERS.yaml` file:
> ```
> Some added maintainers or collaborators do not have sufficient access.
> Error: DBS06 does not have needed access level to zephyrproject-rtos/zephyr
> Error: Process completed with exit code 1.
> ```

Please fill in https://github.com/zephyrproject-rtos/zephyr/issues/new?template=006_nomination.yml in parallel, you should be all set then

--- a/MAINTAINERS.yml
+++ b/MAINTAINERS.yml
@@ -1531,6 +1531,8 @@ Release Notes:
   maintainers:
     - aaronemassey
     - teburd
+  collaborators:
+    - DBS06
   files:
     - drivers/fuel_gauge/
     - dts/bindings/fuel-gauge/

